packages/overlay/src/OverlayTrigger.ts

```diff
@@ -266,6 +269,26 @@ export class OverlayTrigger extends SpectrumElement {
     }
 
     private onTrigger(event: CustomEvent<LongpressEvent>): void {
+        if (
+            event.type === 'mouseleave' &&
+            this.open === 'hover' &&
+            (event as unknown as MouseEvent).relatedTarget ===
+                this.overlaidContent
+        ) {
```

Comment:
not critical, but might be appreciated by future readers:

```suggestion
        const mouseLeavingHoverContent = event.type === 'mouseleave' && this.open === 'hover' && (event as unknown as MouseEvent).relatedTarget === this.overlaidContent;
        
        if (mouseLeavingHoverContent) {
```